curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc2385ff86c76b9248c6eb7.md

```diff
@@ -9,6 +9,15 @@ dashedName: step-5
 
 HTML5 has some elements that identify different content areas. These elements make your HTML easier to read and help with Search Engine Optimization (SEO) and accessibility.
 
+The `main` element is used to represent the main content of the body of an HTML document. Content inside the `main` element should be unique to the document and should not be repeated in other parts of the document.
+
+```html
```

Comment:
This code example should stay because this is the first time campers need to nest elements inside other elements.
And they are not understanding that concept by evidence of the forum posts.
https://forum.freecodecamp.org/search?q=step%205%20%20%23html-css%20cat%20photo%20app%20after%3A2024-03-01

showing them this basic example should alleviate that

Reply:
Yes, but this is an example where we are showing new **syntax**